packages/block-editor/src/components/block-list/block.js

```diff
@@ -172,7 +179,11 @@ function BlockListBlock( {
 	if ( isAligned ) {
 		blockEdit = (
 			<div
-				className="wp-block"
+				className={ `wp-block${
```

Comment:
Tiny nit: rather than concatenating manually, would it be worth calling `classnames( 'wp-block', className )` instead?

Reply:
Ah well spotted! Yes it would 😄